nemo_text_processing/text_normalization/en/taggers/electronic.py

```diff
@@ -92,15 +92,15 @@ def __init__(self, cardinal: GraphFst, deterministic: bool = True):
         protocol = protocol_file_start | protocol_start | protocol_end | (protocol_start + protocol_end)
 
         domain_graph_with_class_tags = (
-            pynutil.insert("domain: \"")
+            pynutil.insert('domain: "')
```

Comment:
Alias all strings as variables

Reply:
I aliased all string literals when appropriate. All variables were placed in `graph_utils.py`.